### ISSUE TITLE
After sending a message in chatbox, a FirebaseError occurs. Solution @ stackoverflow.

### ISSUE THREAD

**Issue by HifiAudi0** [archanaserver/react-messenger-app#1]
- After sending a message in chatbox we are presented with the following error:
FirebaseError
Missing or insufficient permissions.
▶ 15 stack frames were collapsed.
This screen is visible only in development. It will not appear if the app crashes in production.
Open your browser’s developer console to further inspect this error.
This error overlay is powered by `react-error-overlay` used in `create-react-app`.

The following solution seems relevant:
https://stackoverflow.com/questions/65047894/uncaught-error-in-snapshot-listener-firebaseerror-missing-or-insufficient-perm

In summary from above, we must go to the admin console and change database rule to: allow read, write: if true;
